joomla/joomla-cms
# [4.4] Vote : aggregateRating error in Google Rich Results

( see discussion https://github.com/joomla/joomla-cms/discussions/42920) and #42933

### Summary of Changes
When vote plugin is enabled, Rich results are created but are rejected by Google Rich Results tool.


### Testing Instructions
Enable Vote plugin.
Enable Voting in Articles component configuration.
Vote for one or more articles.
Check your article using https://search.google.com/test/rich-results

### Actual result BEFORE applying this Pull Request
Google Rich Results check gives an error 
Review snippets shows 2 errors : 
![rich_snippet_error](https://github.com/joomla/joomla-cms/assets/19435246/facdb827-4df2-4be7-bcc8-f148363fa689)

### Expected result AFTER applying this Pull Request
Google Rich Results check is OK
![rich_snippet_ok](https://github.com/joomla/joomla-cms/assets/19435246/ef3fc2d6-c263-4c8c-a4ec-c46616e1a9c5)

### Link to documentations
Please select:
